### DISCUSSION TITLE
Is Divine Intellect meant to be used only on Llama models?

### DISCUSSION THREAD

**Discussion by jayavanth** [oobabooga/text-generation-webui#5773] [Q&A]
Haven't tried it with models like Mistral but is it supposed to make better or is it intended only for Llama family?

**Comment by TiagoTiago** [comment]
I haven't benchmarked it; but I do have it left as the default even when using Mi**s**tral and Mi**x**tral, and in general no trainwrecks have resulted from that

**Reply by jayavanth** [reply]
Thanks! I've been trying it recently and seems like I get consistent results compared to simple-1. Behaves similarly to when used with Llama2
